ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/gui/PlayGamePanel.java

```diff
@@ -92,6 +93,20 @@ public PlayGamePanel() {
 //		add(resetLabel, constraints);
 //	}
 
+    public static void updateLabels(Result result){
+        int teamID = GameController.getGame().getTurnKeeper().getActiveTeamId() - 1;
```

Comment:
What happens if the team ids are 8 & 9 instead of 1 & 2? We need to be able to support that. We also need to be able to support an arbitrary number of teams, but the UI doesn't need to worry about that right now